feature1/one-more-file.txt

```diff
@@ -1 +1,3 @@
-This file also added in the feature1 branch
\ No newline at end of file
+This file also added in the feature1 branch
```

Comment:
It's really good that you decided to add this line of code

Reply:
Thanks for this comment Priapus